packages/walletconnect/src/index.ts

```diff
@@ -15,6 +15,14 @@ interface WalletConnectOptions {
   connectFirstChainId?: boolean
 }
 
+const isHexString = (value: string | number) => {
```

Comment:
Same method used by ethers project utils package but didn't want to pull in the whole package as a dep.